maven-release-manager/src/main/java/org/apache/maven/shared/release/phase/AbstractRewritePomsPhase.java

```diff
@@ -199,13 +213,18 @@ private void transform(
 
         for (MavenProject project : reactorProjects) {
             URI pom = project.getFile().toURI();
-            logInfo(
-                    result,
-                    "Transforming " + root.relativize(pom).getPath() + ' '
-                            + buffer().project(project.getArtifactId()) + " '" + project.getName() + "'"
-                            + (simulate ? " with ." + getPomSuffix() + " suffix" : "") + "...");
+            final String path = root.relativize(pom).getPath();
+
+            if (exclusionPatterns.stream()
+                    .noneMatch(exclusionPattern -> SelectorUtils.matchPath(exclusionPattern, path))) {
+                logInfo(
```

Comment:
why log this? debug level at most

Reply:
This log is not a newly added code, it only adds exclusion conditions here, Please refer to lines `202-206`.

Reply:
I wouldn't log any of this at info level, and I certainly add more logging to the existing logging at info level.